src/sentry/tasks/clear_expired_rulesnoozes.py

```diff
@@ -0,0 +1,10 @@
+from django.utils import timezone
+
+from sentry.models import RuleSnooze
+from sentry.tasks.base import instrumented_task
+
+
+@instrumented_task(name="sentry.tasks.clear_expired_rulesnoozes", time_limit=65, soft_time_limit=60)
+def clear_expired_rulesnoozes():
+    rule_snooze_ids = RuleSnooze.objects.filter(until__lte=timezone.now()).values_list("id")[:1000]
```

Comment:
@ceorourke why are we only pulling the first 1000 rows?

Reply:
I was basing it off of https://github.com/getsentry/sentry/blob/master/src/sentry/tasks/clear_expired_snoozes.py I assume that's to keep the batch size smaller, but I wasn't totally sure.

Reply:
@ceorourke ok yea makes sense

Reply:
Yeah, this just limits the number of snoozes we delete at once. I added it recently to fix a bug where we were failing to clear up expired snoozes. Technically could have used a ranged queryset as well, but I think this method is fine, since in practice we're not usually deleting more than 1000 snoozes in 5 minutes.